iphone/Classes/TiAppiOSUserNotificationCenterProxy.m

```diff
@@ -47,7 +47,7 @@ - (void)getPendingNotifications:(id)callback
     NSMutableArray *result = [NSMutableArray arrayWithCapacity:[notifications count]];
 
     for (UILocalNotification *notification in notifications) {
-      [result addObject:[TiApp dictionaryWithLocalNotification:notification withIdentifier:nil]];
+      [result addObject:[TiApp dictionaryWithLocalNotification:notification]];
```

Comment:
@janvennemann Everything looks good. I can see warning for [TiApp dictionaryWithLocalNotification:notification]. Reason is  `+ (NSDictionary *)dictionaryWithLocalNotification:(UILocalNotification *)notification` method is not exposed in header file. So we should use `[TiApp dictionaryWithLocalNotification:notification withIdentifier:notification.userInfo[@"id"]]`. 
After fixing this we are good to go.
Thanks!

Reply:
Fixed!